### PR TITLE
Added richards pulled-pork

### PR DESCRIPTION
**Pull request by O5ten**
Richards pulled pork-recept.

### CODE REVIEW

File: pulled-pork/pulled-pork.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- Put in the bottom of the pot
+- Cook as much meatbullion that it covers the onion
+- Put some stubs barbeque sauce and worchestersauce in the pot
+- OPTIONAL: Some garlic
+
+##Slow Cooking
+Put the spices on the tenderloin and put it in a oven secure form and place 
+it in 70-80*c for 10-12 hours for best effect. 
+
+##When done
```
Comment (richasaurec) [inline pulled-pork/pulled-pork.md]: get the meat out of the pot and set aside

File: pulled-pork/pulled-pork.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+It will be alot of fluids so rinse it out so you can throw away the onions. 
+Take 2 forks and shred the meat.
```
Comment (richasaurec) [inline pulled-pork/pulled-pork.md]: use a strainer and a clean pot and pour the fluids and onions into, anything caught in the strainer gets thrown out and the fluids will be saved for later

File: pulled-pork/pulled-pork.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+##Slow Cooking
+Put the spices on the tenderloin and put it in a oven secure form and place 
+it in 70-80*c for 10-12 hours for best effect. 
+
+##When done
+It will be alot of fluids so rinse it out so you can throw away the onions. 
+Take 2 forks and shred the meat. 
+
+##Serving or Freezing
+- If you want to *serve* it now pour much sauce on the pulled pork.
```
Comment (richasaurec) [inline pulled-pork/pulled-pork.md]: when adding the fluids again go easy and use a table spoon. Add a little juice then stir the meat and repeat until desired juicyness is accomplished.

File: pulled-pork/pulled-pork.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Cook as much meatbullion that it covers the onion
+- Put some stubs barbeque sauce and worchestersauce in the pot
+- OPTIONAL: Some garlic
+
+##Slow Cooking
+Put the spices on the tenderloin and put it in a oven secure form and place 
+it in 70-80*c for 10-12 hours for best effect. 
+
+##When done
+It will be alot of fluids so rinse it out so you can throw away the onions.
```
Comment (grahnskij) [inline pulled-pork/pulled-pork.md]: set the meat aside

File: pulled-pork/pulled-pork.md (hunk 2 above)
Comment (grahnskij) [inline pulled-pork/pulled-pork.md:32]: pour fluids and onions through strainer and into a new bowl or pot. Throw away everything in the strainer and save fluids in the pot.

**Pull request merged**